### PR TITLE
[D2IQ-63949] Add release docs for Kafka-2.9.0-2.4.0

### PR DESCRIPTION
**Pull request by rishabh96b**
## Jira Ticket
Resolves https://jira.d2iq.com/browse/D2IQ-63949 and https://jira.d2iq.com/browse/D2IQ-64101

## Description of changes being made
This PR adds the release notes for the release of Kafka-2.9.0-2.4.0

## Priority
- [ ] High
- [x] Medium
- [ ] Low

### CODE REVIEW

**Comment by cmastr** [comment]
@rishabh96b waiting for @SarahStegall to review. then I will merge this.

File: pages/mesosphere/dcos/services/kafka/2.9.0-2.4.0/release-notes/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ render: mustache
 - Updated the scheduler JRE to `v11`.
 
 ### Important Notes
-- Apache Kafka 2.4.0 introduces `[KAFKA-7335] - Store clusterId locally to ensure broker joins the right cluster`, which means that kafka cluster will store the clusterId locally so that it does not join the wrong zookeeper cluster accidentally. Therefore, if your kafka service is connected to default DC/OS zookeeper, changing the zookeeper path is not permitted. Please check [here](https://issues.apache.org/jira/browse/KAFKA-7335) for more information.
+- Apache Kafka 2.4.0 introduces `[KAFKA-7335] - Store clusterId locally to ensure broker joins the right cluster`, which means that a Kafka cluster will store the clusterId locally so that it does not join the wrong Zookeeper cluster accidentally. Therefore, if your Kafkaservice is connected to default DC/OS Zookeeper, changing the Zookeeper path is not permitted. Please check [here](https://issues.apache.org/jira/browse/KAFKA-7335) for more information.
```
Comment (rishabh96b) [inline pages/mesosphere/dcos/services/kafka/2.9.0-2.4.0/release-notes/index.md]: I think we need to change `Kafkaservice` -> `Kafka service`

**Comment by rishabh96b** [comment]
Thanks @SarahStegall  and @cmastr for the approvals. Can we merge this now?

**Comment by cmastr** [comment]
> Thanks @SarahStegall and @cmastr for the approvals. Can we merge this now?

Oh dang I just saw this one. For faster reply, please add readytomerge label and/or ping me in slack. Will merge now.

**Pull request merged**